trin-state/src/lib.rs

```diff
@@ -8,9 +8,10 @@ use trin_core::jsonrpc::launch_trin;
 use trin_core::portalnet::protocol::{PortalEndpoint, PortalnetConfig, PortalnetProtocol};
 
 pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tracing_subscriber::fmt::init();
```

Comment:
what's the reasoning behind removing this from lib.rs and only having it in main.rs?

Reply:
I guess it is usually added in an executable as explained in https://docs.rs/tracing-subscriber/0.2.19/tracing_subscriber/fmt/index.html.
If we leave it in `lib.rs` for `trin-state` and `trin-history`,  it is raising an error if we try to run both networks in parallel from the root crate, because we are trying to initialize `tracing_subscriber` more than once.